sms_sponsorship/templates/sms_registration_confirmation_template.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="sms_registration_confirmation" name="SMS registration confirmation">
+            <t t-call="website.layout">
+                <t t-set="title">Thank you for registration!</t>
+                <div class="oe_structure donation_success">
+                    <div class="thank_you_container">
+                        <div class="thank_you_image_container">
+                            <img class="img-responsive" style="width: 100%" src="/sms_sponsorship/static/src/img/compassion_thankyou_donation.jpg"/>
```

Comment:
You forgot to commit the image. I think as well class img-responsive should be defined somewhere.

Reply:
Image is committed

Reply:
img-responsive is defined in odoo_ext/static/style